backend/app/Http/Controllers/ActivityController.php

```diff
@@ -22,12 +23,15 @@ public function index()
 
     public function show(User $user)
     {
-        $data = $user->activities->latest()->all();
+        $data = collect($user->activities->sortByDesc('created_at'));
 
-        foreach ($data as $key => $activity) {
-            $data[$key]['time_diff'] = Carbon::parse($data[$key]->created_at)->diffForHumans();
+        $new_collection = [];
+
+        foreach ($data as $col) {
+            $col['time_diff'] = Carbon::parse($col->created_at)->diffForHumans();
+            $new_collection[] = $col;
```

Comment:
is this adding items to `$new_collection` ?

Reply:
It is adding a column and adding items to the `$new_collection`. I did this due to a weird behavior of sortByDesc which adds "key" :{Item} to each item instead of just being an array of items

Reply:
I see, you can use `map()` in php